Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1087,11 +1087,11 @@ void RenderGrid::placeItemsOnGrid(std::optional<LayoutUnit> availableLogicalWidt
     performAutoPlacement();
 
 #if ASSERT_ENABLED
-    for (auto* gridItem = currentGrid().orderIterator().first(); gridItem; gridItem = currentGrid().orderIterator().next()) {
-        if (currentGrid().orderIterator().shouldSkipChild(*gridItem))
+    for (auto& gridItem : childrenOfType<RenderBox>(*this)) {
```

Comment:
This *might* be ok since it looks like this is just some debug logic to sanity check grid item placement occurred properly, which should be independent from order, but it probably deserves a justification/mention in the commit message if we want to keep it

Reply:
> This _might_ be ok since it looks like this is just some debug logic to sanity check grid item placement occurred properly, which should be independent from order, but it probably deserves a justification/mention in the commit message if we want to keep it

While this might be ok, cleanup PRs are not supposed to change behavior.

Reply:
makes sense. i have reverted the change on sites where ordering matters